WordPress/src/main/res/layout/quick_start_task_type_item.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout
```

Comment:
>  What if we use ConstraintLayout instead of RelativeLayout to create this layout?

The only reason I didn't convert to `ConstraintLayout` as they were simple layouts and did not require any flattening of the view hierarchy. But I see your point, especially liked this one: `to focus all my energy in getting to know ConstraintLayout better`. I'll go ahead and convert the layouts to `ConstraintLayout`. Hopefully, we standardize this practice 🤞, maybe make it a rule.

Reply:
Thanks @ashiagr , as always, you rock! 🙏 

> Hopefully, we standardize this practice 🤞, maybe make it a rule.

🤞 😃 🤞

Reply:
In afd7312

Reply:
👋 @ashiagr !

Thanks so much for going into this trouble to update all 3 layouts to `ConstraintLayout`, everything works as expected, you rock! 🎸 🙇 💯